### DISCUSSION TITLE
Seeking Collaborators for C++ and website Project - Live Coding Sessions in VS Code -- Just click the link

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135275] [Programming Help]
[[79](https://www.quora.com/answer)
Search Quora
Profile photo for IAmADodo
Where can I post my link to my coding session on VS Code?](https://prod.liveshare.vsengsaas.visualstudio.com/join?17878CBE68A90BAF9A72DB4517576C023748)

**Comment by everforyou** [comment]
Okay, let's do together.

**Reply by ghost** [reply]
VS code isn't giving me any updates. Did you click the link?

**Reply by ghost** [reply]
https://prod.liveshare.vsengsaas.visualstudio.com/join?A421F1C028E696ECF9214771CC9CDBF47889 I accidentally left

**Reply by ghost** [reply]
How come you left? I accidentally left and started a new one for some reason.

**Reply by ghost** [reply]
Are you still going to help me?

**Reply by everforyou** [reply]
Yes, but I'd like to get task instead of fun. :(

**Reply by ghost** [reply]
Join please.

**Reply by ghost** [reply]
Ok, can you please help me write the part of the compiler that translates into the intermediate language that way I am already ahead.

**Comment by ponchanon** [comment]
Work Modality ? Remote ?

**Reply by ghost** [reply]
Remote.
